variables.tf

```diff
@@ -225,3 +231,45 @@ variable "quotas" {
     error_message = "The quota entity must be defined, and at least one of producer_byte_rate or consumer_byte_rate must be set to a non-negative value"
   }
 }
+
+variable "mirroring_enabled" {
```

Comment:
Is there a reason for adding a boolean here instead of just using the value passed in the `mirroring` input to determine if mirroring should be enabled or not? Perhaps its something to do with value not known before apply limitation?